src/build-disk-image.ts

```diff
@@ -94,11 +99,11 @@ export async function buildDiskImage(image: any) {
 
       // Create log folder
       if (!fs.existsSync(selectedFolder)) {
-        await fs.mkdirSync(selectedFolder, { recursive: true });
+        fs.mkdirSync(selectedFolder, { recursive: true });
```

Comment:
might be better with `await fs.promises.mkdir`

Reply:
Switched in new commit.